tasker/validation/task_validator.py

```
@@ -772,6 +772,19 @@ def validate_parallel_task(self, task, task_id, line_number, parallel_tasks):
             )
 
+        # CRITICAL: Parallel blocks cannot use timeout parameter
+        # Parallel blocks coordinate execution but don't run commands themselves
+        if 'timeout' in task:
+            # Concise error for INFO level
+            self.errors.append(
+                f"Line {line_number}: Task {task_id} (parallel) cannot use 'timeout' parameter."
+            )
+            # Detailed explanation for DEBUG level
+            self.debug_log(
+                f"Parallel blocks coordinate execution but don't run commands. "
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)